pkg/fanal/secret/builtin-rules.go

```diff
@@ -792,4 +793,12 @@ var builtinRules = []Rule{
 		SecretGroupName: "secret",
 		Keywords:        []string{"typeform"},
 	},
+	{
+		ID:       "dockerconfig-secret",
+		Category: CategoryDocker,
+		Title:    "Dockerconfig secret exposed",
+		Severity: "HIGH",
+		Regex:    MustCompile(`(?i)(\.(dockerconfigjson|dockercfg):\s*\|*\s*(ey|ew)+)`),
```

Comment:
I think we can upgrade this regex.
If i understand correctly -  secret is base64 encoded string (` YAML files containing dockercfg or dockerconfigjson with base64-encoded secrets.`).
We can add secret group and mask only secret line:
```suggestion
		Regex:           MustCompile(`(?i)(\.(dockerconfigjson|dockercfg):\s*\|*\s*(?P<secret>(ey|ew)+[A-Za-z0-9\/\+=]+))`),
		SecretGroupName: "secret",
```

result before:
```bash
 asdf.txt:2
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   data:
   2 [   *********************JodHRwczovL2luZGV4L

```

result after:
```bash
 asdf.txt:2
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   data:
   2 [   .dockerconfigjson: *********************

```

Reply:
We added `Secret group` (`?P<secret>`). 
Add `SecretGroupName: "secret",` now, please.

Reply:
Added in https://github.com/aquasecurity/trivy/pull/5720/commits/6022117f7336929dd8d97912bb8f6a7cf440a1ad